apps/fortuna/src/command/run.rs

```diff
@@ -143,7 +143,14 @@ pub async fn run(opts: &RunOptions) -> Result<()> {
 
         let provider_info = contract.get_provider_info(opts.provider).call().await?;
         let latest_metadata =
-            bincode::deserialize::<CommitmentMetadata>(&provider_info.commitment_metadata)?;
+            bincode::deserialize::<CommitmentMetadata>(&provider_info.commitment_metadata)
+                .map_err(|e| {
+                    anyhow!(
+                        "Failed to deserialize commitment metadata for chain id {}: {}",
```

Comment:
I have tried to followed the log with chain id like this - https://github.com/pyth-network/pyth-crosschain/blob/main/apps/fortuna/src/keeper.rs#L167

For consistency, can we follow that?
There might be some places where it isn't followed. I am gonna change that.